coding-concepts.js

```diff
@@ -10,46 +10,48 @@
 const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // console.log(colors.push("indigo"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: The .push command as stated by the syllabus should add indigo to the end of the array list.
+// b) Verify and explain: For some reason the code is returning 5. The code is currently showing the index position instead of the word.
+
 
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2023"
 // console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: It should just log the variable set of characters.
+// b) Verify and explain: It logged 10 which is the correct set of characters in the element.
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
 // console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: It's saying to log index position 4, but we have not set up that many index positions, so it should return undefined or a syntax error.
+// b) Verify and explain: The code returned 0 as the answer. Due to the 4th index not being identified the system returned the 4th index value it had setup which was 0.
+
```

Comment:
What it actually returned was "o" which is the fourth character in the string

Reply:
Thank you!